### DISCUSSION TITLE
简的精通堆不起来怎么办家人们

### DISCUSSION THREAD

**Discussion by ll334** [share121/inter-knot#4655] [绝区零]
四百多怎么堆的，我这情况也就局内侃侃三百多
![《绝区零》官方网站-《绝区零》表](https://github.com/user-attachments/assets/bf056e11-ae59-46fe-b14e-78b683c83d7b)
![Screenshot_绝区零_20240908_192514](https://github.com/user-attachments/assets/7f3cb79e-a57f-43c1-a728-390e7ca294e2)

**Comment by tingyu919** [comment]
面板三百多，加个赛斯就够了

**Comment by For-elysia** [comment]
这边建议带一个小草神，大招精通收益很高，然后带一个像钟离一样可以可以带教官套的角色，这样精通就不会差了〔德莉莎—智慧〕〔德莉莎—智慧〕
